modules/@apostrophecms/asset/index.js

```diff
@@ -111,6 +112,8 @@ module.exports = {
         usage: 'Build Apostrophe frontend CSS and JS bundles',
         afterModuleInit: true,
         async task(argv) {
+          // The lock could become huge, cache it, see computeCacheMeta()
+          let _packageLockContentCached;
```

Comment:
In Apostrophe leading `_` means "temporary property not to be stored to the database." To avoid confusion we don't use it for other things (except `_id` since that comes from mongo).